vms/platformvm/txs/executor/standard_tx_executor.go

```diff
@@ -33,8 +35,10 @@ var (
 type StandardTxExecutor struct {
 	// inputs, to be filled before visitor methods are called
 	*Backend
-	State state.Diff // state is expected to be modified
-	Tx    *txs.Tx
+	BlkFeeManager      *commonfees.Manager
```

Comment:
`BlkFeeManager` must cumulate complexity of all transactions in a block.  So we pass it from the StandardTxExecutor caller instead of creating it for each tx